<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WineyardsGamescopeDialog</class>
 <widget class="QDialog" name="WineyardsGamescopeDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manage Gamescope</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop-gaming">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="KTitleWidget" name="titleWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure Gamescope</string>
   </property>
   <property name="comment">
    <string>Configure how Gamescope should manage the window for the games you will run.</string>
   </property>
   <property name="icon">
    <iconset theme="preferences-desktop-gaming">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>481</width>
     <height>475</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="prefsGroup">
    <item>
     <widget class="QLabel" name="gameResTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#gameResTitle {
	background-color: #ECEFF4;
	color: #2E3440;
	border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>Game Resolution</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="gameResBox">
      <item>
       <widget class="QLabel" name="widthLabelGR">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="widthBoxGR">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>640</number>
        </property>
        <property name="maximum">
         <number>3840</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="heightLabelGR">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="heightBoxGR">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>720</number>
        </property>
        <property name="maximum">
         <number>2160</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="gamescopeResTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#gamescopeResTitle {
	background-color: #ECEFF4;
	color: #2E3440;
	border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>Gamescope Resolution</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="gamescopeResBox">
      <item>
       <widget class="QLabel" name="widthLabelGSR">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="widthBoxGSR">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>640</number>
        </property>
        <property name="maximum">
         <number>3840</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="heightLabelGSR">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="heightBoxGSR">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>720</number>
        </property>
        <property name="maximum">
         <number>2160</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="frameRateTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#frameRateTitle {
	background-color: #ECEFF4;
	color: #2E3440;
	border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>Frame Rate Limit</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KComboBox" name="frameRate">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>30</string>
      </property>
      <item>
       <property name="text">
        <string>30</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>45</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>60</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="frameRateNoFocusTitle">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#frameRateNoFocusTitle {
	background-color: #ECEFF4;
	color: #2E3440;
	border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>Frame Rate Limit (No Focus)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KComboBox" name="frameRateNoFocus">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>30</string>
      </property>
      <item>
       <property name="text">
        <string>30</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>45</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>60</string>
       </property>
       <property name="icon">
        <iconset theme="time">
         <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="integerScaling">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Use integer scaling?</string>
      </property>
      <property name="icon">
       <iconset theme="time">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="windowTypeTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#windowTypeTitle {
	background-color: #ECEFF4;
	color: #2E3440;
	border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>Window Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KComboBox" name="windowType">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Borderless</string>
       </property>
       <property name="icon">
        <iconset theme="window">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fullscreen</string>
       </property>
       <property name="icon">
        <iconset theme="window">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>590</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset theme="delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
